pkgs/by-name/co/composefs/package.nix

```diff
@@ -74,7 +79,7 @@ stdenv.mkDerivation (finalAttrs: {
     description = "File system for mounting container images";
     homepage = "https://github.com/containers/composefs";
     changelog = "https://github.com/containers/composefs/releases/tag/v${finalAttrs.version}";
-    license = with lib.licenses; [ gpl3Plus lgpl21Plus ];
+    license = with lib.licenses; [ gpl2Only asl20 ];
```

Comment:
changelog says it's gpl2Plus not gpl2Only

Reply:
See the PR description. The licence change applies to MOST of the code, but there is still some code with other licences. (presumable gpl2Plus is when you use it as a library, not any of the tooling)